### PR TITLE
Rework how c2patool is configured

### PR DESCRIPTION
**Pull request by gpeacock**
## Changes in This Pull Request
35b54e5: Rework c2patool configuration

## Checklist
- [ ] This PR represents a single feature, fix, or change.
- [ ] All applicable changes have been documented.
- [ ] Any `TO DO` items (or similar) have been entered as GitHub issues and the link to that issue has been included in a comment.

### CODE REVIEW

**Review by scouten-adobe — CHANGES_REQUESTED** [review]
A couple of quick comments here and ~~I'd also like to make some proofreading and style changes in this PR. Will do after meetings (likely after 1pm Pacific).~~

File: Cargo.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [workspace]
-members = ["sdk", "c2patool"]
+members = ["sdk", "c2patool", "make_tests"]
```
[the remark this answers is not in the document]
Comment (gpeacock) [reply Cargo.toml:2]: I'm not crazy about a three part command line name. I actually think make_tests is very descriptive - it makes tests.

[the remark this answers is not in the document]
Comment (gpeacock) [reply Cargo.toml:2]: It could be make_assets or make_images

[the remark this answers is not in the document]
Comment (scouten-adobe) [reply Cargo.toml:2]: IMHO it doesn't make the _entire_ test; of the above, I'd probably be most happy with make_assets.

**Comment by scouten-adobe** [comment]
Based on Slack discussion just now, I'm withdrawing my request to edit doc comments in c2patool.

I still would like to see the `make_tests` name changed.

**Review by scouten-adobe — APPROVED** [review]

**Pull request merged**